### DISCUSSION TITLE
How do you remove the link to the Budibase installation at the top of emails sent via automation?

### DISCUSSION THREAD

**Discussion by FlaminWrap** [Budibase/budibase#4626] [Help] [answered]
Hi Guys,

How do you remove the link to the Budibase installation at the top of emails sent via automation? I was wanting to send a confirmation email to clients if they have provided a valid email address and have it match our company branding.

![image](https://user-images.githubusercontent.com/97764630/155107638-599cd186-4144-44a5-9d0b-8745f78c406b.png)

**Comment by mjashanks** [comment, marked as answer]
You can edit the `Base format` from the `Email` section on the main platform...

![image](https://user-images.githubusercontent.com/3524181/155111582-bb8cce16-1785-4458-8cce-4bb0545ddc5a.png)


This is the section you need to edit:

![image](https://user-images.githubusercontent.com/3524181/155111749-8bca5558-0a1f-47b5-b03f-21bff8c7e3c8.png)

**Reply by FlaminWrap** [reply]
Thank you
